bokehjs/src/lib/models/axes/axis.ts

```diff
@@ -309,9 +305,9 @@ export class AxisView extends GuideRendererView {
       let val: number
 
       if (side == "above" || side == "below")
-        val = w*s + (h/hscale)*c
+        val = w*s + h*c
       else
-        val = w*c + (h/hscale)*s
+        val = w*c + h*s
```

Comment:
this empirical scale value was added because the basic expected computation did not yield correct visual results when tick labels were rotated. Would love to jettison it, but what has changed such that is is no longer needed?

Reply:
If I recall correctly, last time this was discussed, the conclusion was this isn't needed, thus removal. No tests failed after this change. If you have any test case that can validate this code, then I can revise this change.

Reply:
I don't recall that but will take your word